Makefile

```diff
@@ -7,17 +7,18 @@ export $(shell sed 's/=.*//' ./.env)
 export NODE_ENV ?= development
 
 
-.PHONY: build-local api-local web-local print-env run-local
+.PHONY: build api-local web-local print-env run-local
```

Comment:
What's this .PHONY thing?

Reply:
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
Cool, over my head 😄